pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -444,7 +444,9 @@ public void start() throws PulsarServerException {
             this.initWaiter.countDown();
             this.started = true;
         } catch (Exception ex) {
+            log.error("Failed to start the extensible load balance.", ex);
```

Comment:
How about merging these two logs into one?

Reply:
+1

Reply:
Thanks for the suggestion. Merged.